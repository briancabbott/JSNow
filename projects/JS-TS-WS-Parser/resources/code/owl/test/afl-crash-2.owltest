---expr =
 'y'* 'x' : a
 'x' : b
