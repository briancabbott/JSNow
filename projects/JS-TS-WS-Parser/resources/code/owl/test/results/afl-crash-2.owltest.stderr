error: this grammar is ambiguous

. x 

  can be parsed in two different ways: as

. x      
  expr:a 

  or as

. x      
  expr:b 

